webpack/assets/javascripts/react_app/components/Bookmarks/BookmarksActions.js

```diff
@@ -22,13 +18,9 @@ export const getBookmarks = (url, controller) => {
   return _getBookmarks(uri.toString(), controller);
 };
 
-export const modalOpened = query => ({
-  type: BOOKMARKS_MODAL_OPENED,
+export const setQuery = query => ({
+  type: BOOKMARKS_SET_QUERY,
```

Comment:
see my comment in `Bookmarks.js`, I feel this action is redundant and we should remove it.

Reply:
@laviro Agree, removed.  (Originally this was the `BOOKMARKS_MODAL_OPENED` action and it set the query as a side effect in the reducer (bad!)  So I'm glad it went on this journey :laughing: )